python/triton/runtime/autotuner.py

```diff
@@ -117,6 +122,10 @@ def kernel_call():
             self.post_hook(args)
 
         try:
+            if self.use_cuda_graph:
+                with torch.cuda.stream(self.benchmarkig_stream):
+                    bench_res = do_bench_cudagraph(kernel_call, rep=self.num_reps, return_mode="median")
+                return [bench_res]
```

Comment:
I can probably drop the brackets and it'll still works (even though other branches of this function return a list of floats). Let me try that

Reply:
Also, the default mode should be `mean`?

Reply:
Removed the brackets. Regarding the return mode: for non-cudagraphed benchmarking `do_bench` returns a tuple `(median, quantile at 20%, quantile at 80%)`, and then the best config is found by sorting these tuples. So I've decided to use median for cudagraphed benchmarking. Alternatively, I can also make it exactly as in non-cudagraphed case (i.e. return a tuples of three quantiles), or switch to mean, or make the return mode configurable - let me know what's best. Hopefully CUDA-graphed kernel runtimes are stable enough so that all these options would give close results